activerecord/lib/active_record/insert_all.rb

```diff
@@ -29,6 +31,11 @@ def updatable_columns
       keys - readonly_columns - unique_by_columns
     end
 
+    def primary_keys
+      Array(model.primary_key)
+    end
+
+
```

Comment:
Are this double blank lines intentional?

Reply:
Yes! 😊